### ISSUE TITLE
[$15] - List view of past challenges (Mobile view)

### ISSUE THREAD

**Issue by birdofpreyru** [appirio-tech/topcoder-app#1048]
Here are screenshot of my Past Challenges in my Dashboard, when viewed in Mobile size, with the list view enabled:
![scr-12](https://cloud.githubusercontent.com/assets/20144632/23160496/0405b89a-f827-11e6-92c2-8b24f681a001.png)
![scr-13](https://cloud.githubusercontent.com/assets/20144632/23160544/38213672-f827-11e6-9176-a9bab9ea45db.png)
As you see, in some cases the gray rectangle in the right part of the card goes beyond the card margins. This should be fixed.

**Comment by birdofpreyru** [comment]
@Colinh84 In respect of the original problem, it looks fine now, thanks. Though, I see another problem now: some tiles tend to go into a second column:
![scr-09](https://cloud.githubusercontent.com/assets/20144632/23860685/f4326652-0807-11e7-8102-4c108179e614.png)

Are you interested to fix it for extra $10? No problem, if you are not, I can accept this as it is, and open a new ticket for that fix.

**Comment by Colinh84** [comment]
@birdofpreyru ok no problem, I will work on this extra problem here.

**Comment by Colinh84** [comment]
@birdofpreyru I have not been able to reproduce this column issue. I have tested in both Chrome and Firefox in both responsive and mobile emulation modes, and I can never see more than 1 column, even with my real topcoder account with hundreds of past challenges showing.. Can you give more information on what browser and resolution you saw this bug? Also just to make sure, did you clear your browser cache before testing?

**Comment by birdofpreyru** [comment]
@Colinh84 I've double checked, after cleaning the cache, and still see the issue. Tested with my real Topcoder account, on Chrome/Ubuntu with responsive emulation mode. Here screencast: https://youtu.be/vWcwKGG2bVw As you may note, the issue is visible not on all mobile sizes, only in some range.

**Comment by Colinh84** [comment]
@birdofpreyru I tried again to test, using the same environment and browser you mentioned and still cannot reproduce. I even modified the challenge details to semi-randomly display listed challenges of all types (since I initially didn't have any design or data-science challenges listed) and with different roles to catch all markup and styling differences between challenge listings. Below I included a screencast of my testing. Also to note, I did the testing with the branch applied to the latest dev, so I updated the original PR for this issue rebased to newest dev, maybe that is what makes the difference?
 
![screen](https://cloud.githubusercontent.com/assets/2287621/24032852/b74b4368-0b1c-11e7-927f-147c2d2641d5.gif)
